base/util_test.go

```diff
@@ -1735,3 +1735,23 @@ func TestCASToLittleEndianHex(t *testing.T) {
 	littleEndianHex := Uint64CASToLittleEndianHex(casValue)
 	require.Equal(t, expHexValue, string(littleEndianHex))
 }
+
+func TestUint64CASToLittleEndianHexAndStripZeros(t *testing.T) {
+	hexLE := "0x0000000000000000"
+	u64 := HexCasToUint64(hexLE)
+	hexLEStripped := Uint64ToLittleEndianHexAndStripZeros(u64)
+	u64Stripped := HexCasToUint64(hexLEStripped)
```

Comment:
I might be overlooking something, but I expected this line to be using HexCasToUint64ForDelta.  Am I misunderstanding how Uint64ToLittleEndianHexAndStripZeros works?

Reply:
Good spot, test didn't have sufficient test cases so wasn't failing. Good catch, added a new test case and fixed this.